src/status_im/contexts/centralized_metrics/tracking.cljs

```diff
@@ -43,23 +43,46 @@
     :screen/onboarding.syncing-progress
     :screen/onboarding.syncing-progress-intro
     :screen/onboarding.syncing-results
-    :screen/onboarding.welcome})
+    :screen/onboarding.welcome
+
+    ;; Collectibles
+    :screen/wallet.collectible})
 
 (defn track-view-id-event
   [view-id]
   (when (contains? view-ids-to-track view-id)
     (navigation-event (name view-id))))
 
-(defn tracked-event
-  [[event-name second-parameter]]
-  (case event-name
-    :profile/get-profiles-overview-success
-    (user-journey-event app-started-event)
+(defn collectibles-fetched-event
+  [has-collectibles?]
+  (key-value-event "collectibles-fetched" :has-collectibles has-collectibles?))
+
+(defn navigated-to-collectibles-tab-event
+  [location]
+  (key-value-event "navigated-to-collectibles-tab" :location location))
```

Comment:
for key-value-event func i prefer @clauxx suggestion, as for the task i'm working i need to send more data and we can do it without breaking other usage 
```clojure 
(defn key-value-event
  [event-name & args]
  (if (odd? (count args))
    (throw (ex-info "Expected even number of args"
                    {:args   args
                     :n-args (count args)}))
    {:metric
     {:eventName  event-name
      :platform   platform/os
      :appVersion build/app-short-version
      :eventValue (apply hash-map args)}}))
```

Reply:
@mohsen-ghafouri What are the benefits of using a var args signature to mimic a map and build it manually with `hash-map` if we can pass a map structure directly as in the diff above? Also there would be no need to check the args are pairs because the compiler would guarantee that. `throw` we should almost never use because it breaks the experience for the user.

Reply:
Your suggestion make sense, I applied in my PR

Reply:
Thanks @mohsen-ghafouri 🚀